app/resources/assets/sass/_location_map_report.scss

```diff
@@ -86,6 +86,14 @@ This is where to put SCSS for the location report that shows a map.
 					white-space: nowrap;
 					max-width: 100%;
 				}
+
+				em {
+					margin-right: 5px;
+				}
+
+				p {
+					margin-bottom: 0px;
```

Comment:
When specifying 0 of any unit, you can drop the unit to have slightly cleaner code.  In more specific terms, replace with ```margin-bottom: 0;```